notifications/src/main/java/com/dojo/notifications/model/SlackMethodsLookupByEmail.java

```diff
@@ -0,0 +1,46 @@
+package com.dojo.notifications.model;
+
+import com.hubspot.slack.client.methods.JsonStatus;
+import com.hubspot.slack.client.methods.MethodWriteMode;
+import com.hubspot.slack.client.methods.RateLimitingTier;
+import com.hubspot.slack.client.methods.RateLimitingTiers;
+import com.hubspot.slack.client.methods.SlackMethod;
+
+public enum SlackMethodsLookupByEmail implements SlackMethod {
+    users_lookupByEmail(MethodWriteMode.READ, RateLimitingTiers.TIER_3, JsonStatus.ACCEPTS_JSON)
```

Comment:
Please leave one blank line under class declaration and format code with Ctrl+Alt+L